packages/components/checkbox/src/index.ts

```diff
@@ -0,0 +1,3 @@
+export { default as Checkbox } from './Checkbox';
```

Comment:
You are only allowed to export default one time in each file.

Should export like this:
```suggestion
export { Checkbox } from './Checkbox';
export { CheckboxGroup } from './CheckboxGroup';
```